### PR TITLE
Add sidekiq throttle and fix url parmeters bug

### PR DESCRIPTION
**Pull request by sgelbart**

### CODE REVIEW

File: config/initializers/sidekiq.rb [hunk 1]
```diff
@@ -10,8 +10,8 @@
 
 # Create oscn throttling strategy
 Sidekiq::Throttled::Registry.add(:oscn, {
-  threshold: { limit: ENV.fetch('OSCN_THROTTLE', 240).to_i, period: 1.minute },
-  concurrency: { limit: ENV.fetch('OSCN_CONCURRENCY', 10).to_i }
+  threshold: { limit: ENV.fetch('OSCN_THROTTLE', 6).to_i, period: 1.minute },
+  concurrency: { limit: ENV.fetch('OSCN_CONCURRENCY', 1).to_i }
```
Comment (sgelbart) [inline config/initializers/sidekiq.rb:14]: I noticed this had actually already been implemented. We actually want both this and slowweb until we can update slowweb. Slowweb works for things that aren't in workers but we'll have to use this for workers for now.

File: app/services/importers/new_court_case.rb [hunk 2]
```diff
@@ -8,8 +8,7 @@ def initialize(link)
       @case_number = link.text
       @case_types = CaseType.oscn_id_mapping
       @counties = County.pluck(:name, :id).to_h
-      @uri = URI(link['href'])
-      @params = CGI.parse(uri.query)
+      @params = CGI.parse(URI(link['href']).query).map { |k,v| [k.downcase, v.first] }.to_h
```
Comment (sgelbart) [inline app/services/importers/new_court_case.rb:11]: There was a bug where casemasterID could be in the URL as camelcase or all lowercase.

File: app/services/importers/new_court_case.rb [hunk 3]
```diff
@@ -24,11 +23,11 @@ def perform
     private
 
     def oscn_id
-      params['casemasterID'].first.to_i
+      params.fetch("casemasterid")
```
Comment (sgelbart) [inline app/services/importers/new_court_case.rb:26]: using `fetch` throws an exception if the value isn't there

**Review by holdenmitchell — APPROVED** [review]
👍

**Pull request merged**